includes/class-gp-route-translation-helpers.php

```diff
@@ -39,11 +39,25 @@ public function __construct() {
 	 * @return void
 	 */
 	public function original_permalink( $project_path, $original_id, $locale_slug = null, $translation_set_slug = null, $translation_id = null ) {
+		$original = GP::$original->get( $original_id );
+		if ( ! $original ) {
+			$this->die_with_404();
+		}
 		$project = GP::$project->by_path( $project_path );
 		if ( ! $project ) {
 			$this->die_with_404();
 		}
 
+		if ( $project->id !== $original->project_id ) {
+			$project = GP::$project->get( $original->project_id );
+			wp_redirect( self::get_permalink( $project->path, $original_id, $locale_slug, $translation_set_slug, $translation_id ) );
```

Comment:
The function `get_permalink` allows and uses 4 parameters. I think we should remove the fifth parameter here `$translation_id` since it's unused.

Reply:
Indeed, thanks!